src/sentry/tasks/reports.py

```diff
@@ -160,30 +161,30 @@ def merge_series(target, other, function=operator.add):
     return results
 
 
+def __get_range_chunked(issue_ids, start, stop, rollup):
```

Comment:
```suggestion
def _get_range_chunked(issue_ids, start, stop, rollup):
```

This can read a little confusing out of a class context because name mangling isn't applied at the module level. There's not really any benefit here over a single underscore to indicate that it's not intended as public API.